views/punchIn.js

```diff
@@ -1,5 +1,29 @@
+var acc = document.getElementsByClassName("accordion");
+var i;
+
+for (i = 0; i < acc.length; i++) {
+  acc[i].addEventListener("click", function () {
+    this.classList.toggle("active");
+    var panel = this.nextElementSibling;
+    if (panel.style.display === "block") {
+      panel.style.display = "none";
+    } else {
+      panel.style.display = "block";
+    }
+  });
+}
+
+function spinner() {
+  console.log("in here");
```

Comment:
dont push console log statements, they are only for debugging purposes and printing catch block errors